opni_inference_service/opnilog_trainer.py

```diff
@@ -38,6 +41,15 @@
     use_ssl=True,
 )
 
+RETRY_LIMIT = 5
+
+
+def mask_logs(all_training_logs):
```

Comment:
we probably don't need another function for this simple loop, just do a list comprehension